Makefile

```diff
@@ -38,15 +38,13 @@ BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:$(IMAGE_TAG)
 
 # CHANNELS define the bundle channels used in the bundle.
 # Add a new line here if you would like to change its default config. (E.g CHANNELS = "candidate,fast,stable")
-ifneq ($(origin CHANNELS), undefined)
+CHANNELS ?= stable
```

Comment:
I've been banging my head with this change (discussed offline)...

Starting at [v0.9.0](https://github.com/Kuadrant/authorino-operator/releases/tag/v0.9.0), we define one OLM channel called `stable`. It used to be `alpha` in previous versions and, until this PR gets merged, it still is in the main branch, out of which we build the bundle and catalog `:latest` images.

On one hand, having the `stable` channel for the released versions as well as in main makes things easy and straightforward. It leaves `alpha`completely in the past and ensures a more seamless UX between released and latest versions.

On the other hand, I wonder if semantically we might be sending the wrong message that _latest == stable_, and therefore if `stable` should be reserved to released versions of the operator only.

To exemplify, consider the following. Say you have:

```yaml
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: operatorhubio-catalog
  namespace: authorino-operator
spec:
  sourceType: grpc
  image: quay.io/kuadrant/authorino-operator-catalog:v0.9.0
  displayName: Authorino Operator
---
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: authorino-operator
spec:
  channel: stable
  installPlanApproval: Automatic
  name: authorino-operator
  source: operatorhubio-catalog
  sourceNamespace: authorino-operator
```

And then, in the `CatalogSource`, you change to:

```yaml
  image: quay.io/kuadrant/authorino-operator-catalog:latest
```

**What would you expect to happen?** With `stable` in the released version and `alpha` in main/latest, the “upgrade” from v0.9.0 to latest would fail. Whereas, with `stable`/`stable`, latest is rollout out without hiccups.

**What’s the safest behaviour to enforce here?** Would failing the upgrade from a released version to latest be a valid way to tell users that latest is **not** to be considered stable? Or, as a user, would you prefer to keep it simple with one single channel name and make the upgrade work out of the box?